lib/workers/repository/init/vulnerability.ts

```diff
@@ -200,11 +200,8 @@ export async function detectVulnerabilityAlerts(
           matchFileNames,
         };
 
-        if (
-          datasource === PypiDatasource.id &&
-          normalizePythonDepName(depName) !== depName
-        ) {
-          matchRule.matchPackageNames?.push(normalizePythonDepName(depName));
+        if (datasource === PypiDatasource.id) {
+          matchRule.matchPackageNames = [normalizePythonDepName(depName)];
```

Comment:
I think this it should be removed, but I don't remember at this point. And I could miss it during cherry-pick.